pdns/opensslsigners.cc

```diff
@@ -226,11 +235,13 @@ class OpenSSLRSADNSCryptoKeyEngine : public DNSCryptoKeyEngine
   void convertToPEM(std::FILE& outputFile) const override;
 
   [[nodiscard]] storvector_t convertToISCVector() const override;
+
+  // TODO Fred: ::hash() can probably be completely removed.
```

Comment:
Isn't `hash()` used in `makeDSFromDNSKey()`, at least?

Reply:
Yes, but it's really a hacky way to use DCKEs for hashing.
So when comes the time to remove support for OpenSSL1.1, we can re-implement `makeDSFromDNSKey()` properly. I created #12464 to track some of the `TODO`s.

Reply:
That works for me, although I'm unfortunately pretty sure it will take years before we can remove OpenSSL 1.1 support :-(